content/en/service_management/events/correlation/patterns.md

```diff
@@ -36,28 +49,31 @@ To create or adjust a pattern-based correlation:
     : The max duration that net new alerts will be added to a case 
 
     Deduplicate events for those alerts for
-    : The max duration current alerts which have been correlated, and continue to flap or have not resolved will be reviewed for, before opening new case....
-
-    {{< img src="service_management/events/correlation/pattern/timeframe.png" alt="correlation timeframe explanation, see how alerts fold into a case" style="width:90%;" >}}
+    : The max duration for current alerts which have been correlated, but continue to flap or have not resolved, will be deduped to the alert in the existing case before opening new case
```

Comment:
I'm not sure if my suggestion makes sense here. Just trying to break this sentence up.

```suggestion
    : The max duration for current alerts which have been correlated, but continue to flap or have not resolved. Alerts are deduped to the first alert in the existing case before opening a new case
```